test/command.bats

```diff
@@ -54,3 +54,8 @@ load helpers
 	run ! "$CRIO_BINARY_PATH" --log-size-max 18446744073709551616
 	[[ "$output" == *"value out of range"* ]]
 }
+
+@test "devicemapper is deprecated" {
+	"$CRIO_BINARY_PATH" -s devicemapper
```

Comment:
You probably need to use "run", like the test above this one :
```suggestion
	run ! "$CRIO_BINARY_PATH" -s devicemapper
```

This will run the command and expect an error, and it will set the "ouput" variable that you're testing below.

With the current line, the test fails because this call causes an error:
```
06:14:38 # time="2023-06-08 04:14:38.073069018Z" level=warning msg="devicemapper storage driver is deprecated and will be removed in cri-o v1.28"
06:14:38 # time="2023-06-08 04:14:38.073268619Z" level=debug msg="[graphdriver] trying provided driver \"devicemapper\"" file="drivers/driver.go:335"
06:14:38 # time="2023-06-08 04:14:38.073347519Z" level=error msg="Failed to GetDriver graph devicemapper /var/lib/containers/storage" file="drivers/driver.go:308"
06:14:38 # time="2023-06-08 04:14:38.073400820Z" level=fatal msg="validating root config: failed to get store to set defaults: failed to GetDriver graph devicemapper /var/lib/containers/storage: driver not supported" file="crio/main.go:448"
```

Reply:
I don't think we're compiling with devmapper support, hence this failure. we actually do expect to be able to specify it now but ci isn't set up to do so